src/main/java/roomescape/web/controller/ReservationController.java

```diff
@@ -31,15 +33,15 @@ public ResponseEntity<List<ReservationResponse>> getReservations() {
 	}
 
 	@PostMapping
-	public ResponseEntity<ReservationResponse> create(@RequestBody ReservationRequest request) {
-		ReservationRequest.validateReservation(request);
-		return ResponseEntity.ok().body(this.reservationService.create(request));
+	public ResponseEntity<ReservationResponse> create(@RequestBody ReservationRequest request,
+			LoginMember loginMember) {
+		return new ResponseEntity<>(this.reservationService.create(request, loginMember), HttpStatus.CREATED);
```

Comment:
> 200 으로만 내려주던 것들을 201, 204 등으로 세분화시켰어요

👍 👍 👍 
추가로, `ResponseEntity.created()` 를 사용해보는건 어떨까요?
그리고 CREATED 응답을 한다면 어떤 것을 응답에 함께 넣어주면 좋을지도 한 번 확인해보면 좋을 것 같아요.
예를 들어 아래와 같이 만들어볼 수 있겠네요 : 

```suggestion
        var response = this.reservationService.create(request, loginMember);
        var uri = URI.create("/reservations/" + response.id());
        return ResponseEntity.created(uri).body(response);
```